jarbas/chamber_of_deputies/views.py

```diff
@@ -30,6 +30,11 @@ def get(self, request):
         values = map(self.request.query_params.get, params)
         filters = {k: v for k, v in zip(params, values) if v}
 
+        # filter cpf or cnpj without dot and dash
+        hasCfpCnpj = self._bool_param('cnpj_cpf')
+        if hasCfpCnpj is not None:
+            self.queryset = self.queryset.cnpj_cpf(request.query_params.get('cnpj_cpf'))
```

Comment:
You use the search fix here, but the admin site still shows the bug, as documented in the issue. You need to hook this special case to the admin site too, not only in the API.